.github/workflows/p2p-workflow-prod.yaml

```diff
@@ -25,11 +21,39 @@ on:
         required: false
         type: string
         default: '.'
+      registry-path:
+        required: false
+        type: string
+        default: 'prod'
+      version:
+        required: false
+        type: string
+        default: ''
+      image-name:
+        required: true
+        type: string
 
 
 jobs:
+  get-latest-version:
```

Comment:
this shouldn't be here, we can't assume that the version for prod comes from a registry